### PR TITLE
SAK-40166 Allow rubric grading in Questions screen (Samigo)

### PR DESCRIPTION
**Pull request by bgarciaentornos**
- It requires to pass a new parameter (evaluatedItemId) all around the Rubrics structure to difference between users on the same screen.
- Each Rubric grading is a modal component, cloned the first time its called (a workaround to make CKeditor editable and avod ids clashing).

### CODE REVIEW

**Comment by mpellicer** [comment]
@bgarciaentornos would you mind to review the PR and fix the accessLint issues?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java [hunk 1]
```diff
@@ -103,6 +104,8 @@
 	private static final String noAnswer = ContextUtil.getLocalizedString(MSG_BUNDLE, "no_answer");
 	private static final String noneOfTheAbove = ContextUtil.getLocalizedString(MSG_BUNDLE, "none_above");
 
+	private RubricsService rubricsService = ComponentManager.get(RubricsService.class);
```
Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java]: Use JSF's managed bean DI vs the static cover here.

Comment (bgarciaentornos) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java]: Sorry, I'm not sure about the precise dependency injection way you want here. Samigo beans are always doing it like this (not the most desirable, I concur) and I thought consistency within the tool could have priority against introducing different paths. Anyway, let me know your proposal and I'll fix it. And maybe we can create a jira to substitute all the other similar cases. Thanks!

**Comment by bgarciaentornos** [comment]
I have sorted those accessslint issues, although they were all related to hidden fields. Plus all the other Rubrics input fields are still unlabeled though...

**Comment by mpellicer** [comment]
@bgarciaentornos Resolve the conflict please, thanks!

**Comment by bgarciaentornos** [comment]
Done!

**Pull request merged**